arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json

```diff
@@ -2130,11 +2096,35 @@
         },
         "backupManagementType": {
           "description": "Type of backup managmenent for the job. Possible value: AzureIaasVM.",
```

Comment:
Yeah, you don't want that `Possible Value` text there, it's not consistent with your enum values below anyway.